### PR TITLE
Add about Nadesh Seen

### PR DESCRIPTION
**Pull request by nadeshseen**
Details of Nadesh Seen

### CODE REVIEW

**Review by illseption — APPROVED** [review]
@divyam3897 Approved for merge.

File: submissions/nadeshseen/about.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# About Me
+My name is Nadesh Seen. I am CSE student at Panjab University, India.
```
Comment (gurpreetsingh00885) [inline submissions/nadeshseen/about.md]: I am **a** CSE student ...

File: submissions/nadeshseen/about.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+# Experience
+I am Web Developer in Design and Innovation Group, Chandigarh, India.
```
Comment (gurpreetsingh00885) [inline submissions/nadeshseen/about.md]: I am **a** web ...

**Review by gurpreetsingh00885 — CHANGES_REQUESTED** [review]

**Comment by gurpreetsingh00885** [comment]
A suggestion - never ever work on master branch. Create a new branch for every issue you work on.

**Pull request closed without merging**